tck/src/main/java/com/hedera/hashgraph/tck/methods/sdk/param/CommonTransactionParams.java

```diff
@@ -0,0 +1,40 @@
+package com.hedera.hashgraph.tck.methods.sdk.param;
+
+import com.hedera.hashgraph.tck.methods.JSONRPC2Param;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+/**
+ * CommonTransactionParams (part of AccountCreateParams) for SDK client
+ */
+@SuppressWarnings("OptionalUsedAsFieldOrParameterType")
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+public class CommonTransactionParams extends JSONRPC2Param {
```

Comment:
Not used, but might be useful in the future. Can be made abstract and be inherited.